cross_compile/sysroot_compiler.py

```diff
@@ -192,9 +192,9 @@ def __init__(
         self._target_sysroot = self._cc_root_dir / SYSROOT_DIR_NAME
         self._ros_ws_directory = self._target_sysroot / self._ros_workspace_dir
         self._qemu_directory = self._target_sysroot / QEMU_DIR_NAME
-        self._dockerfile_directory = Path(__file__).parent / DOCKER_WS_NAME
+        self._dockerfile_directory = Path(__file__).parent / ROS2_DOCKERFILE_NAME
```

Comment:
If we are planning to have separate dockerfiles, then instead of using the constant, should this now be an argument that is passed in to this constructor based on which system is being built?

Reply:
That's how I envision doing it. That change will be part of the next PR. Since this change is just to rename the dockerfile, I'm not making any other interface changes here.